src/lib/components/healthselector/__snapshots__/Healthselector.component.test.js.snap

```diff
@@ -0,0 +1,282 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Would be better to separate the test into a specific commit and make sure to update the test every time change the code. 

I updated the test and pushed a commit^^

Reply:
Ok will do for next time :) Thanks for updating